C/math/PrimeNumber.c

```diff
@@ -0,0 +1,29 @@
+#include <stdio.h>
+int main() {
+   int num, flag = 0;
+   
+   printf("Enter number to check prime or not : ");
+   scanf("%d",&num);
+   
+   for(int i=2 ; i < num/2 ; i++) {
```

Comment:
We need to check for factors of the number only till ```sqrt(n)```. Refer [this](https://stackoverflow.com/questions/5811151/why-do-we-check-up-to-the-square-root-of-a-prime-number-to-determine-if-it-is-pr) to learn more.